{"name": "Негативная проверка создания 2-ух одинаковых курьеров", "status": "passed", "description": "Запрос должен вернуть ошибку и правильный код ответа", "start": 1721633265743, "stop": 1721633266467, "uuid": "004ee81f-7a05-4d8b-8306-78dd8d551df3", "historyId": "a1c163e6329341bce654e28c1cebd101", "testCaseId": "a1c163e6329341bce654e28c1cebd101", "fullName": "tests_create_courier.test_creating_2_identical_couriers.TestCreate2IdenticalCouriers#test_сreate_2_identical_couriers_error", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('reg')"}, {"name": "parentSuite", "value": "tests_create_courier"}, {"name": "suite", "value": "test_creating_2_identical_couriers"}, {"name": "subSuite", "value": "TestCreate2IdenticalCouriers"}, {"name": "host", "value": "DESKTOP-IKUPUFD"}, {"name": "thread", "value": "13948-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests_create_courier.test_creating_2_identical_couriers"}]}